BAZProject/MVVM/Controllers/HomeController/DetailMovieView.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  DetailMovieView.swift
+//  BAZProject
+//
+//  Created by victor manzanero on 09/11/22.
+//
+
+import UIKit
+
+class DetailMovieView: UIViewController {
```

Comment:
Marca tus clases como final
```suggestion
final class DetailMovieView: UIViewController {
```